tests/security/tpm2/tpm2_env_setup.pm

```diff
@@ -62,10 +63,14 @@ EOF
     $self->select_serial_terminal;
 
     # Start the emulator
-    assert_script_run "su - tss -c '/usr/lib/ibmtss/tpm_server&'";
+    if (is_sle) {
+        assert_script_run "su - tss -c '/usr/lib/ibmtss/tpm_server&'";
+    } else {
+        assert_script_run "su - tss -c '/usr/libexec/ibmtss/tpm_server&'";
+    }
 
     # Start the tpm2-abrmd service
```

Comment:
Start -> Restart

Reply:
Forgot to modify the comments as well lol, will do